### DISCUSSION TITLE
Remote Virtual Desktop

### DISCUSSION THREAD

**Discussion by mygule** [rustdesk/rustdesk#4169] [General]
May I ask if it currently supports the controlled end without a display? If not, will it be supported in the future

**Comment by silmaril42** [comment]
Yes it does.

I am currently doing some tests with a second Windows PC that has no display and is usually accessed via RDP.

Connecting RustDesk first shows the lock screen you would see on the display. Logging in a user disconnects the RDP session, just as it does when doing this at a display.

Switching screen resolutions is working as expected, too.

**Comment by rustdesk** [comment]
Yes in nightly build https://github.com/rustdesk/rustdesk/releases/tag/nightly
## Windows
https://github.com/rustdesk/rustdesk/pull/4082, virtual display driver will be installed when you install ruustdesk nightly version
## Linux
Need more steps, https://github.com/rustdesk/rustdesk/wiki/Headless-Linux-Support
## Mac
Not ready yey
